pyexif/__init__.py

```diff
@@ -277,7 +277,11 @@ def setTags(self, tags_dict):
             raise TypeError('tags_dict is not instance of dict')
         vallist = []
         for tag in tags_dict:
-            vallist.append("-{0}='{1}'".format(tag, tags_dict[tag]))
+            val = tags_dict[tag]
+            # escape double quotes in case of string type
+            if isinstance(val, basestring):
```

Comment:
`basestring` is a Python2-specific class. Instead, use `six.string_types` to support both Python2 and Python3